### DISCUSSION TITLE
Page parser did not find next page URL

### DISCUSSION THREAD

**Discussion by karpatxnegen** [kevinzg/facebook-scraper#403] [General]
Hi,

**Page parser did not find next page URL - [page_iterators.generic_iter_pages(facebook_scraper.page_iterators) line:95]**
This error is from our application log.
Maybe something is changed?

Thanks,
István

**Comment by neon-ninja** [comment]
This occurs when the page has no further posts

**Reply by karpatxnegen** [reply]
I'm sorry, but I don't think so. These are well known, very active group pages

**Reply by neon-ninja** [reply]
What groups?
